gandalf/src/main/java/io/github/btkelly/gandalf/holders/DialogStringsHolder.java

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright 2016 Bryan Kelly
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.github.btkelly.gandalf.holders;
+
+import android.content.Context;
+import android.support.annotation.StringRes;
+
+import io.github.btkelly.gandalf.R;
+
+/**
+ * Holds strings that will be displayed in the AlertDialog built by BootstrapDialogUtil
+ */
+public class DialogStringsHolder {
+
+    private Context context;
+    private String updateAvailableTitle;
+    private String updateRequiredTitle;
+    private String alertTitle;
+    private String downloadUpdateButton;
+    private String skipUpdateButton;
+    private String closeAppButton;
+    private String okButton;
+
+    public DialogStringsHolder(Context context) {
+        this.context = context;
+    }
+
+    public String getUpdateAvailableTitle() {
+        if (updateAvailableTitle == null) {
```

Comment:
I just checked the `isBlank()` code. If I use it, the user will not be able to set an empty title (empty String). Even if this seems to be silly, I think we should let the user make whatever he wants. Do you think we could stay with a "simple null-check" here ?